### PR TITLE
Move iOS view management to compose.ui

### PR DESCRIPTION
**Pull request by elijah-semyonov**
## Proposed Changes

Copy SkikoUIView from skiko, use local version instead.

## Testing

Test: TBD

## Depends on
https://github.com/JetBrains/skiko/pull/757

### CODE REVIEW

File: compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt [hunk 1]
```diff
@@ -0,0 +1,566 @@
+package org.jetbrains.skiko.bridge
```
Comment (MatkovIvan) [inline compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt:1]: Shouldn't it be placed into separate module for window management? or be a part of androidx.*

**Review by MatkovIvan — CHANGES_REQUESTED** [review]
Blocking it for now. By splitting skiko we meant creating a separate module for window management

File: compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt (hunk 1 above)
Comment (elijah-semyonov) [reply compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt:1]: Discussed in internal slack channel, see Dima's recent question.

File: compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+import platform.UIKit.*
+import platform.darwin.NSInteger
+import kotlin.math.max
+import kotlin.math.min
+import org.jetbrains.skiko.*
+
+@Suppress("CONFLICTING_OVERLOADS")
+@ExportObjCClass
+@OptIn(InternalSkikoApi::class)
+class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol {
```
Comment (dima-avdeev-jb) [inline compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt]:
```suggestion
internal class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol {
```

File: compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt (hunk 1 above)
Comment (dima-avdeev-jb) [reply compose/ui/ui/src/uikitMain/kotlin/org/jetbrains/skiko/bridge/SkikoUIView.kt:1]: move to `package androidx.compose.ui.platform`

**Pull request closed without merging**